showcase/BUILD.bazel

```diff
@@ -19,7 +19,8 @@ proto_library_with_info(
         "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto",
 #        "//showcase/gapic-showcase-extended/proto:showcase_proto_extended",
         "@com_google_googleapis//google/cloud:common_resources_proto",
-        "@com_google_googleapis//google/cloud/location:location_proto"
+        "@com_google_googleapis//google/cloud/location:location_proto",
+        "@com_google_googleapis//google/iam/v1:iam_policy_proto"
```

Comment:
So adding this `iam_policy_proto` implicitly enables IAM feature? Is this how we are suppose to enable IAM?

Reply:
I believe having this in the bazel rule enables it: https://github.com/googleapis/gapic-showcase/blob/b146f892d98ea0d733c45a2315941e8e6ab69c90/schema/google/showcase/v1beta1/showcase_v1beta1.yaml#L29

But this was to generate the Java related Mixin code.